### PR TITLE
S07 chargepoint calendar modal

### PR DESCRIPTION
**Pull request by SebSamPro**
Fix style about charpointCalendar, chargepointBook 1 and 2

### CODE REVIEW

**Review by Bravax — APPROVED** [review]

**Review by ItsMatDev — APPROVED** [review]

File: frontend/src/components/ChargepointBook/ChargepointBook.jsx [hunk 1]
```diff
@@ -17,68 +17,71 @@ const station = {
   accessibility: "PMR",
 };
 function ChargepointBook() {
+  const { setopenBooking } = useStore();
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChargepointBook/ChargepointBook.jsx]: camelCase please

File: frontend/src/components/ChargepointBook/ChargepointBook.jsx [hunk 2]
```diff
@@ -17,68 +17,71 @@ const station = {
...
-            </button>
-          </Dialog.Close>
-        </Dialog.Content>
-      </Dialog.Portal>
-    </Dialog.Root>
+    <>
+      <h2 className="stationName">{station.name}</h2>
+      <section className="stationInfos">
+        <p className="date">
+          Date : <span className="dateChoose">{station.date}</span>
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChargepointBook/ChargepointBook.jsx]: time tag will be better here

File: frontend/src/components/ChargepointBook/ChargepointBook.jsx [hunk 3]
```diff
@@ -17,68 +17,71 @@ const station = {
...
+        </p>
+        <p className="time">
+          Horaire : <span className="timeChoose">{station.time}</span>
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChargepointBook/ChargepointBook.jsx]: here too

File: frontend/src/components/ChargepointBook2/ChargepointBook2.jsx [hunk 4]
```diff
@@ -1,64 +1,57 @@
...
-            pauseOnFocusLoss
-            draggable
-            pauseOnHover
-            theme="light"
-          />
-        </Dialog.Content>
-      </Dialog.Portal>
-    </Dialog.Root>
+    <main className="ChargepointBook2TextContainer">
+      <header className="ChargepointBook2Text">
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChargepointBook2/ChargepointBook2.jsx]: P tag will be better here

File: frontend/src/components/ChargepointBook2/ChargepointBook2.jsx [hunk 5]
```diff
@@ -1,64 +1,57 @@
...
+        Un tarif de 2 euros vous sera facturés pour la réservation.
+      </header>
+      <section className="beCareful">
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChargepointBook2/ChargepointBook2.jsx]: here too

File: frontend/src/components/ChargepointBook2/ChargepointBook2.jsx [hunk 6]
```diff
@@ -1,64 +1,57 @@
...
+        Les frais de réservation ne seront pas remboursés en cas d'annulation.
+      </section>
+      <section className="ChargepointBook2Text">
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChargepointBook2/ChargepointBook2.jsx]: and here

File: frontend/src/components/ChargepointCalendar/ChargepointCalendar.jsx [hunk 7]
```diff
@@ -66,85 +66,91 @@ function ChargepointCalendar() {
...
+          <legend className="selectDate">Choisir une date :</legend>
+          <Calendar
+            className="calendar"
+            onChange={handleDateChange}
+            value={selectedDate}
+          />
+          {selectedDate && (
+            <>
+              <p className="confirmDate">Votre date sélectionnée :</p>
+              <span className="selectedDateDisplay">
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChargepointCalendar/ChargepointCalendar.jsx]: time tag will be better

File: frontend/src/components/ChargepointCalendar/ChargepointCalendar.jsx [hunk 8]
```diff
@@ -66,85 +66,91 @@ function ChargepointCalendar() {
...
+                {selectedDate.toLocaleDateString()}
+              </span>
+            </>
+          )}
+        </fieldset>
+
+        {generateTimeSlots()[0] ? (
+          <article className="timeVehicule">
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChargepointCalendar/ChargepointCalendar.jsx]: Why an article in a form ?

**Review by Yoann-Fortin — DISMISSED** [review]

**Review by JulienOza — APPROVED** [review]

File: frontend/src/components/ChargepointBook/ChargepointBook.jsx [hunk 9]
```diff
@@ -17,68 +17,71 @@ const station = {
...
+      <h2 className="stationName">{station.name}</h2>
+      <section className="stationInfos">
+        <p className="date">
+          Date : <time className="dateChoose">{station.date}</time>
+        </p>
+        <p className="time">
+          Horaire : <time className="timeChoose">{station.time}</time>
+        </p>
+        <p className="adressStation">
+          Adresse :{" "}
```
Comment (fantinerudent) [inline frontend/src/components/ChargepointBook/ChargepointBook.jsx]: you can delete all {" "}

File: frontend/src/components/ChargepointBook2/ChargepointBook2.jsx [hunk 10]
```diff
@@ -1,64 +1,57 @@
...
-            draggable
-            pauseOnHover
-            theme="light"
-          />
-        </Dialog.Content>
-      </Dialog.Portal>
-    </Dialog.Root>
+    <main className="ChargepointBook2TextContainer">
+      <p className="ChargepointBook2Text">
+        Un tarif de 2 euros vous sera facturés pour la réservation.
```
Comment (fantinerudent) [inline frontend/src/components/ChargepointBook2/ChargepointBook2.jsx]: facturé

File: frontend/src/components/ChargepointCalendar/ChargepointCalendar.jsx [hunk 11]
```diff
@@ -66,85 +66,91 @@ function ChargepointCalendar() {
...
+              <option value="">Sélectionnez votre véhicule</option>
+              {user.vehicle.map((vehicle) => (
+                <option key={user.id} value={vehicle}>
+                  {vehicle}
+                </option>
+              ))}
+            </select>
+          </time>
+        ) : (
+          <p>Pas de créneaux disponibles</p>
```
Comment (fantinerudent) [inline frontend/src/components/ChargepointCalendar/ChargepointCalendar.jsx]: pas de créneau disponible

**Review by fantinerudent — CHANGES_REQUESTED** [review]

File: frontend/src/components/StationMiniCard/StationMiniCard.jsx [hunk 12]
```diff
@@ -12,6 +14,7 @@ function StationMiniCard({ stations }) {
     chademo,
   };
 
+  const { sethandleModal, SetopenBooking } = useStore();
```
Comment (fantinerudent) [inline frontend/src/components/StationMiniCard/StationMiniCard.jsx]: setHandleModal, setOpenBooking non?

**Review by fantinerudent — CHANGES_REQUESTED** [review]

**Review by fantinerudent — APPROVED** [review]

**Pull request merged**